cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -68,3 +73,16 @@ func ValidateTokenDiscovery(c *kubeadm.TokenDiscovery, fldPath *field.Path) fiel
 	allErrs := field.ErrorList{}
 	return allErrs
 }
+
+func ValidateServiceSubnet(subnet string, fldPath *field.Path) field.ErrorList {
```

Comment:
@luxas I agree we should add tests. The fact there are no tests are another reason to do so!